src/robot/libraries/Process.py

```diff
@@ -333,19 +335,19 @@ def __init__(self):
 
     def run_process(
         self,
-        command,
-        *arguments,
-        cwd=None,
-        shell=False,
-        stdout=None,
-        stderr=None,
-        stdin=None,
-        output_encoding="CONSOLE",
-        alias=None,
-        timeout=None,
-        on_timeout="terminate",
-        env=None,
-        **env_extra,
+        command: str,
+        *arguments: "str | Secret",
+        cwd: "str | None" = None,
+        shell: bool = False,
+        stdout: "str | None" = None,
+        stderr: "str | None" = None,
+        stdin: "str | Path | Secret | Any" = None,
```

Comment:
I believe `stdin` could be `int` and then `Any` wouldn't be needed (but `None` would also need to be added).

Reply:
changed to 
```
        stdin: "str | Path | Secret | int | None" = None,
```